app/assets/javascripts/timeline.js

```diff
@@ -96,6 +96,21 @@ samson.filter("userFilter",
   }
 );
 
+samson.filter("timeDateFilter",
+  function() {
+    return function(td, timeFormat) {
+      if (timeFormat == 'local') {
+        return moment(td).format('LLL');
+      } else if (timeFormat == 'utc'){
+        return moment(td).utc().format();
+      } else {
+        return moment(td).fromNow();
+      }
+
```

Comment:
unnecessary newline 
can it be `else if == now ... else throw "Unknown format xyz"` to make future debugging easier